### ISSUE TITLE
libphonon.so.4

### ISSUE THREAD

**Issue by oberon-manjaro** [mysms/mysms-desktop-app#1]
I am trying to get mysms to work on Archlinux and am confronted with `libphonon.so.4: cannot open shared object file: No such file or directory`
By default this library is placed in **/usr/lib/**
Since I see that the Debian **phonon** package places it in **/usr/lib/x86_64-linux-gnu/** I have placed a symlink there, but that doesn't seem to help.
Could you kindly tell me where **mysms** is expecting the library?
Thanks

**Comment by notz** [comment]
You shoud look in debian package control file https://github.com/mysms/mysms-desktop-app/blob/master/debian/control

Their you see that you need:

```Build-Depends: debhelper (>= 8.0.0),
               libqt4-dev (>> 4.7.0),
               libqtwebkit-dev,
               libphonon-dev

```

So you need qt 4 & libqtwebkit & libphonon dev packages for compilation.

It's a shared library so usr/lib should be fine.

Cou can try with "ldd mysms" if all required libs are installed 
```

**Comment by oberon-manjaro** [comment]
Thanks for your reply.
I had tried to substitute the dependencies but I had missed **qtwebkit**.
It's now working fine with the binary from [here](http://download.opensuse.org/repositories/home:/mysms/Debian_7.0)
Is there maybe a newer binary around? Or would you know of a way to compile the git-sources directly on Arch?

**Comment by notz** [comment]
You have to write a pkgbuild file. Should be easy to adapt a existing file from qt4 app.

**Comment by oberon-manjaro** [comment]
Well, my problem is that no makefile is supplied... I've uploaded a PKGBUILD to the AUR using the .deb packages mentioned above.That's satisfactory in any case :)

**Comment by oberon-manjaro** [comment]
Can I ask which commands you are using to compile this for Debian?

**Comment by oberon-manjaro** [comment]
Now I get `Unknown module(s) in QT: multimedia`
Installed dependencies I have: 'gcc' 'libstdc++296' 'qtwebkit' 'phonon-qt4' 'qt4'
the only thing I can find is **qt5-multimedia** but nothing the likes for qt4 ...

**Comment by oberon-manjaro** [comment]
ok. building with **qmake-qt4** solves that ;)

**Comment by oberon-manjaro** [comment]
But now I get `../mysms-desktop-app/webview.cpp:77:45: error: ‘class QUrl’ has no member named ‘fileName’
     QString defaultFileName = request.url().fileName();`

**Comment by notz** [comment]
Yes, that's a compatibility bug. The latest source code we build with qt5. Their is no linux (deb) release, only a windows release. But it should work also with qt5 (don't know if their are problems with new qt5 versions)
